fedgov-cv-map-services-proxy/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+google.map.api.key=<google-api-key>
```

Comment:
Make sure to add a note to remove the google api key from the private resources package and also update the ReadMe file in the develop branch to add the google API key here.